src/components/Languages.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import generateKey from '@tammiec/generatekey';
+
+const Languages = ({ data, ...props }) => {
```

Comment:
Why the spread on the props? Are we going to use it in some future or they're for console logging?